avm/res/service-fabric/cluster/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.26.54.24096",
-      "templateHash": "2708589601167951653"
+      "version": "0.25.53.49325",
```

Comment:
Hey @lsnoddy, please update bicep cli to the latest version

Reply:
Updated bicep cli to latest version